src/components/molecules/Table/Toolbar/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Button } from '@chakra-ui/react';
+import { FilterColumn, FilterRuleDefinition } from 'model/table/table-filter-types';
+import SearchBar from 'components/molecules/Table/SearchBar';
+import Filter from 'components/molecules/Table/Filter';
+import { Link as RouterLink } from 'react-router-dom';
+
+export interface ToolbarProps {
+  showButton?: boolean;
+  showFilter?: boolean;
+  showSearchBar?: boolean;
+  buttonText?: string;
+  buttonRoutePath?: string;
+  filterColumns?: FilterColumn[];
+  onSearch?: (value: string) => void;
+  onFilter?: (filterRules: FilterRuleDefinition[]) => void;
+}
+
+const Toolbar = ({
+  showButton,
+  showFilter,
+  showSearchBar,
+  buttonText,
+  filterColumns,
+  buttonRoutePath,
+  onSearch,
+  onFilter,
+}: ToolbarProps) => (
+  <Box mb="0.5rem" pb="0.5rem" display="flex" borderBottom="1px solid #e2e8f0" flexDirection="row">
+    {showButton && buttonRoutePath && (
+      <Button as={RouterLink} to={buttonRoutePath} mx="5px">
+        {buttonText}
+      </Button>
+    )}
+    {showFilter && filterColumns && onFilter && <Filter columns={filterColumns} onFilter={onFilter} />}
```

Comment:
I don't like this implementation, I think it can bite us in the ass in the future. Again, `showFilter` is not passed at all from the components above. What is the logic behind this?

Of course, I can see defaulting these props, but I don't understand the logic.

Reply:
During initial implementation (couple weeks ago) I wanted to make this components more generic, but as you already pointed - we always pass `true` as default prop.
I removed unnecessary props from those implementations as you suggested.